src/gameData/GuiConsts.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2005 - 2015 Settlers Freaks (sf-team at siedler25.org)
+//
+// This file is part of Return To The Roots.
+//
+// Return To The Roots is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// Return To The Roots is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
+
+#ifndef GUICONSTS_H_
+#define GUICONSTS_H_
+
+#include <boost/array.hpp>
+
+const boost::array<float, 5> ZOOM_FACTORS = { 0.5f, 0.75f, 1.f, 1.25f, 1.5f};
```

Comment:
If you include that in another cpp file that does not use this, you get an unused warning. This does not get generated for C-arrays though.
Oh and you need double-braces  ` ... SUPPRESS_UNUSED ZOOM_FACTORS = {{ ... }};` as the array is a subobject :/ -> https://travis-ci.org/Return-To-The-Roots/s25client/jobs/147873685